backend/core/src/main/java/org/sonarsource/sonarlint/core/analysis/AnalysisService.java

```diff
@@ -419,15 +419,15 @@ private ServerActiveRule tryConvertDeprecatedKeys(String connectionId, ServerAct
       var templateRuleKeyWithCorrectRepo = RuleKey.parse(ruleOrTemplateDefinition.getKey());
       var ruleKey = new RuleKey(templateRuleKeyWithCorrectRepo.repository(), ruleKeyPossiblyWithDeprecatedRepo.rule()).toString();
       return new ServerActiveRule(ruleKey, possiblyDeprecatedActiveRuleFromStorage.getSeverity(), possiblyDeprecatedActiveRuleFromStorage.getParams(),
-        ruleOrTemplateDefinition.getKey());
+        ruleOrTemplateDefinition.getKey(), new ArrayList<>());
```

Comment:
You could add a TODO for all these cases that should be covered by another ticket of the iteration